src/components/Nav/Nav copy.js

```diff
@@ -0,0 +1,147 @@
+import React, { useState, useEffect } from 'react';
+import styled, { css } from 'styled-components/macro';
+import { Link, useLocation } from 'react-router-dom';
+
+function Nav() {
+  const [crrLocation, setCrrLocation] = useState();
+  const location = useLocation();
+
+  useEffect(() => {
+    setCrrLocation(location.pathname);
+  }, [location.pathname]);
+
+  useEffect(() => {
+    console.log(crrLocation);
+  }, [crrLocation]);
+
+  return (
+    <Container>
+      {location.pathname === '/' ? (
+        <LinkGroup>
+          <Link to="/">
+            <i class="fas fa-bars"></i>
+            <span>프로젝트 둘러보기</span>
+          </Link>
+          <Link to="/project-start">프로젝트 올리기</Link>
+        </LinkGroup>
+      ) : null}
+      <Link to="/">
+        <Logo alt="logo" src="/images/logo.png" />
+      </Link>
+      {location.pathname === '/' ? (
+        <SearchAndLogin>
+          <button type="button">
+            <i class="fas fa-search"></i>
+          </button>
+          <Link to="/signin">
+            <span>로그인 / 회원가입</span>
+            <UserProfile />
+          </Link>
+        </SearchAndLogin>
+      ) : null}
+    </Container>
+  );
+}
+
+const Container = styled.div`
+  ${({ theme }) => theme.flexSet('space-between')};
+  padding: 1rem 0;
+  ${({ theme }) => theme.desktop`
+        ${({ theme }) => theme.flexSet()};
+        width: 1032px;
+        padding: 0.3rem 0;
+        margin: 0px auto;
+      `};
+`;
+
+const LinkGroup = styled.div`
+  ${({ theme }) => theme.flexSet()}
+  font-size: 1rem;
+  font-weight: 700;
+  width: fit-content;
+
+  a {
+    ${({ theme }) => theme.flexSet()};
+    padding: 0px 1rem;
+    color: ${({ theme }) => theme.colors.black};
+    line-height: 100px;
+
+    &:hover {
+      color: ${({ theme }) => theme.colors.secondaryGray};
+    }
+
+    i {
+      position: relative;
+      bottom: 2px;
+      margin-right: 4px;
+    }
+
+    span {
+      display: none;
+      ${({ theme }) => theme.desktop`
+          display: inline`};
+    }
+
+    &:nth-of-type(2) {
+      display: none;
+      ${({ theme }) => theme.desktop`
+          display: flex`};
+    }
+  }
+`;
+
+const Logo = styled.img`
+  width: 70px;
+  margin: 1.5rem;
+  /* ${({ theme }) => theme.posCenter()}; */
+  ${({ theme }) => theme.desktop`
+         width: 85px;
+        `};
```

Comment:
```suggestion
  ${({ theme }) => theme.desktop`
    width: 85px;
  `};
```
저한테는 indent가 전반적으로 이상하게 보이네요 확인해주세요

Reply:
오.. 제 눈에도 이상해보입니다. 아마 제가 손으로 indent를 넣은 것 같습니다